app/routes/apply/eligibility/claim/about-child.js

```diff
@@ -33,17 +33,17 @@ module.exports = function (router) {
 
     try {
       const child = new AboutChild(
-        req.body.FirstName,
-        req.body.LastName,
-        req.body['dob-day'],
-        req.body['dob-month'],
-        req.body['dob-year'],
-        req.body['child-relationship']
+        req.body?.FirstName,
+        req.body?.LastName,
+        (req.body && req.body['dob-day']) ?? '',
```

Comment:
Haven't tried, but I *think* you can just do `req.body?.['dob-day'] ?? ''` for this and the other similar ones.